utils_for_test.go

```diff
@@ -46,9 +46,9 @@ func OutboundConnection(call *OutboundCall) (*Connection, net.Conn) {
 func GetConnections(ch *Channel) []*Connection {
 	var connections []*Connection
```

Comment:
i just merged a change into master earlier today which removes this function :) can you rebase off latest master, then you won't need any change here.